deckhouse-controller/crds/module-update-policy.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: moduleupdatepolicies.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: external-module-manager
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: moduleupdatepolicies
+    singular: moduleupdatepolicy
+    kind: ModuleUpdatePolicy
+    shortNames:
+      - mup
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          description: |
+            Defines the update settings for a module's release.
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - update
+                - moduleReleaseSelector
```

Comment:
Probably, the module release selector can be absent if an administrator wants to completely disable this update policy.

Reply:
I think we will add something like `mode: Never` lately, to disable the policy